.azure/export.json

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
Next Step: Wait for the export workflow run
 if the run failed, check diff file, if succeed, resolve this comment and merge the pull request
 if you have any problem, please file an [issue](https://github.com/Azure/Edge-infrastructure-quickstart-template/issues/new/choose) or contact arcIaCSupport@microsoft.com